public/app/features/search/components/DashboardListPage.tsx

```diff
@@ -39,9 +40,15 @@ export const DashboardListPage: FC<Props> = memo(({ navModel, match, location })
     });
   }, [match.params.uid]);
 
+  const [testState, setTestState] = useState(false);
```

Comment:
Need to remove this - it's here to test my sanity that checkboxes actually work 😅